contact.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link href="styles-contact.css" rel="stylesheet" />
+    <title>Contact me.</title>
+  </head>
+
+  <body>
+    <div class="navbar">
+      <a href="index.html">Work</a>
+      <a href="about.html">About</a>
+      <a class="active" href="contact.html">Contact</a>
+    </div>
+    <br />
```

Comment:
you could give the `contact` section a `margin-top` or `padding-top` to create breathing room